packages/teleport/src/components/hooks/useUrlFiltering/getQueryParams.ts

```diff
@@ -24,11 +24,13 @@ export default function getResourceUrlQueryParams(
   const search = searchParams.get('search');
   const sort = searchParams.get('sort');
 
+  const sortParam = sort ? sort.split(':') : null;
+
```

Comment:
i think if we move `labelClick` to the file where it is used, this file can also be moved to where it is only used and delete this file

i would always try to define the func closest to where it is used, and only separate it if it needs to be used else where (we have a utils file `teleport/lib/util`)